scripts/install.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+set -e
+
+OCDEV_VERSION="latest"
+GITHUB_RELEASES_URL="https://github.com/redhat-developer/ocdev/releases/download/v0.0.1"
+BINTRAY_URL="https://dl.bintray.com/ocdev/ocdev/latest"
+
+# CANCELLATION_TIMEOUT specifies the number of seconds the script will wait
+# for the user to cancel the execution before continuing on.
+CANCELLATION_TIMEOUT=15
+INSTALLATION_PATH="/usr/local/bin/"
+PRIVILEGED_EXECUTION="sh -c"
+
+DEBIAN_GPG_PUBLIC_KEY="https://bintray.com/user/downloadSubjectPublicKey?username=bintray"
+DEBIAN_MASTER_REPOSITORY="https://dl.bintray.com/ocdev/ocdev-deb-dev"
+DEBIAN_LATEST_REPOSITORY="https://dl.bintray.com/ocdev/ocdev-deb-releases"
+
+RPM_MASTER_YUM_REPO="https://bintray.com/ocdev/ocdev-rpm-dev/rpm"
+RPM_LATEST_YUM_REPO="https://bintray.com/ocdev/ocdev-rpm-releases/rpm"
+
+SUPPORTED_PLATFORMS="
+darwin-amd64
+linux-amd64
+linux-arm
+windows-amd64
+"
+
+echo_stderr ()
+{
+    echo "$@" >&2
+}
+
+command_exists() {
+	command -v "$@" > /dev/null 2>&1
+}
+
+check_platform() {
+    kernel="$(uname -s)"
+    if [ "$(uname -m)" = "x86_64" ]; then
+        arch="amd64"
+    fi
+
+    platform_type="${kernel,,}-$arch"
```

Comment:
this line is still failing o macOS,

```
▶ ./install.sh
# Starting ocdev installation...
# Detecting distribution...
./install.sh: line 43: ${kernel,,}-$arch: bad substitution
```
will fix it tomorrow, before we merge this